### DISCUSSION TITLE
Javascript methods in manual

### DISCUSSION THREAD

**Discussion by marenostrum1981** [django-commons/django-unicorn#331] [Ideas]
Hello, happy new year.
Maybe i'm missing something that is already said.
I was trying to call a method from javascript (i'm using sweetalert for confirmations).
I know there is a call() method for call javascript directly from the component in the UnicornView.

But i was inspecting the javascript "unicorn.js" and I saw few nice methods, like Unicorn.call('component', 'method', ...args)

With that I can use perfectly django-unicorn with confirmation button on Sweetalerts.

It would be great to have these methods on the docs.

Thank you.

```
<button class="btn btn-primary " onclick="swal.fire({ 
        icon: 'warning',
        title: '¿Save whatever?',
        showCancelButton: true,
        confirmButtonText: 'Confirm',
        cancelButtonText: 'Cancel',
        }).then((result) => {
        if (result.isConfirmed) {
            Unicorn.call('my_component', 'save')
            swal.fire('Saved correctly', '', 'success');
        } 
        })">
            Save
        </button>
```

**Comment by marenostrum1981** [comment]
> Hello, happy new year. Maybe i'm missing something that is already said. I was trying to call a method from javascript (i'm using sweetalert for confirmations). I know there is a call() method for call javascript directly from the component in the UnicornView.
> 
> But i was inspecting the javascript "unicorn.js" and I saw few nice methods, like Unicorn.call('component', 'method', ...args)
> 
> With that I can use perfectly django-unicorn with confirmation button on Sweetalerts.
> 
> It would be great to have these methods on the docs.
> 
> Thank you.
> 
> ```
> <button class="btn btn-primary " onclick="swal.fire({ 
>         icon: 'warning',
>         title: '¿Save whatever?',
>         showCancelButton: true,
>         confirmButtonText: 'Confirm',
>         cancelButtonText: 'Cancel',
>         }).then((result) => {
>         if (result.isConfirmed) {
>             Unicorn.call('my_component', 'save')
>             swal.fire('Saved correctly', '', 'success');
>         } 
>         })">
>             Save
>         </button>
> ```

It's here, sorry. 
https://www.django-unicorn.com/docs/actions/#calling-methods

**Reply by adamghill** [reply]
Glad you found it in the docs! Let me know if there is another place that would make sense to mention this functionality. 👍
